specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json

```diff
@@ -1344,7 +1344,7 @@
             "type": "string",
             "description": "The expand expression to apply on the operation.",
             "enum": [
-              "instanceView"
+              "InstanceView"
```

Comment:
I can see that there are breaking changes in enum values. The casing has been changed and the api-version has also not been bumped. However @ravbhatnagar  from ARM team has signed off over [here](https://github.com/Azure/azure-rest-api-specs-pr/pull/149). Hence I am signing off as well.